src/pages/member/UserHome.js

```diff
@@ -1,15 +1,173 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { Helmet } from 'react-helmet';
+import Typography from '@material-ui/core/Typography'
+import EventCard from './EventCard'
+import { fetchBackend, updateEvents } from "../../utils";
+import { COLOR } from '../../constants/Constants'
+import House from '../../assets/house.svg'
+import Card from '@material-ui/core/Card'
+import CardContent from '@material-ui/core/CardContent'
+import { makeStyles } from '@material-ui/core/styles'
+import { connect } from "react-redux";
+
+const styles = {
+    home: {
+        color: COLOR.TITLE_GREEN,
+        fontStyle: 'normal',
+        fontWeight: 'bold',
+        paddingLeft: '85px',
+        paddingTop: '50px',
+    },
+    page: {
+        height: '100vh'
+    },
+    container: {
+        display: 'flex'
+    },
+    header: {
+        fontStyle: 'normal',
+        fontWeight: 'bold',
+        fontSize: '36px',
+        paddingLeft: '76px',
+        paddingTop: '45px'
+    },
+    reward: {
+        fontStyle: 'normal',
+        fontWeight: 'normal',
+        fontSize: '22px',
+        paddingLeft: '76px',
+        paddingTop: '15px',
+    },
+    house: {
+        position: 'absolute',
+        left: '685px',
+        top: '110px'
+    }
+}
+
+const useStyles = makeStyles({
```

Comment:
you should pick either `styles` object or to use `makeStyles()`, don't use both in one file